### ISSUE TITLE
This is really nice, but is there an option for Firefox?

### ISSUE THREAD

**Issue by Tenome** [laxyapahuja/bunken#6]
I stopped using Chrome recently because it was bugging out on me a lot. Just wondering if this or a similar script is available. Thanks.

Edit: Copy-pasting the main .js seems to work, in case anyone else stumbles upon this. I also recommend adjusting line 89 so it only has the title and ISBN, the author gave me irrelevant results every now and then.

> fetch(`${API}${source}?title=${encodeURIComponent(bookTitle)}&isbn=${encodeURIComponent(ISBNCode)}`).then(response => {

**Comment by laxyapahuja** [comment]
Exactly. There's a script that gets injected into Goodreads webpage. As long as you have that script, you can make it run for any browsers out there. But there's an extension called TamperMonkey that makes injecting scripts easier to manage for Firefox. I'll look into releasing an official release for Firefox soon.

As for the author thing, the API is set up according to this extension only. So, the author name information is required by the API if it doesn't end up with any results just by searching the ISBN code. It's not needed but it's better you keep it just in case when you're browsing an obscure book.
